app/views/media_objects/_item_view.html.erb

```diff
@@ -115,6 +117,63 @@ Unless required by applicable law or agreed to in writing, software distributed
       }
     });
     currentPlayer = avalonPlayer.player;
+
+    var $video = currentPlayer.container;
```

Comment:
This is a convention used in some projects. The $ in front is used to indicate that the thing points to a jQuery object, rather than some other thing.